pkg/objectstore/s3/s3.go

```diff
@@ -82,5 +84,21 @@ func GetObjLockInfo(backupLocation *stork_api.BackupLocation) (*s3.GetObjectLock
 		}
 		return nil, err
 	}
-	return out, err
+	objLockInfo := &common.ObjLockInfo{}
+	if (out != nil) && (out.ObjectLockConfiguration != nil) {
+		if aws.StringValue(out.ObjectLockConfiguration.ObjectLockEnabled) == "Enabled" {
+			objLockInfo.LockEnabled = true
+		}
+		if out.ObjectLockConfiguration.Rule != nil &&
+			out.ObjectLockConfiguration.Rule.DefaultRetention != nil {
+			objLockInfo.LockMode = aws.StringValue(out.ObjectLockConfiguration.Rule.DefaultRetention.Mode)
+			objLockInfo.RetentionPeriodYears = aws.Int64Value(out.ObjectLockConfiguration.Rule.DefaultRetention.Years)
+			objLockInfo.RetentionPeriodDays = aws.Int64Value(out.ObjectLockConfiguration.Rule.DefaultRetention.Days)
+		} else {
+			//This is an invalid object-lock config, no default-retention but object-loc enabled
+			objLockInfo.LockEnabled = false
+			return nil, fmt.Errorf("only object lock is enabled but no retention policy is set on the bucket")
```

Comment:
Can we please remove that word **_"only"_** from the front, it makes the error statement little informal. or Please reword it if you think it can be rewritten differently.

Reply:
Fixed.